conda-store-server/conda_store_server/server/app.py

```diff
@@ -28,6 +28,9 @@
 )
 from traitlets.config import Application, catch_config_error
 
+logger = logging.getLogger("app")
+logger.setLevel(logging.INFO)
```

Comment:
This needs to be updated to use `basicConfig` because this uses the implicit handler.